### PR TITLE
Python 3 migration.

### PR DESCRIPTION
**Pull request by mrgambal**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,6 @@
 requirements = [str(ir.req) for ir in install_reqs]
 test_requirements = requirements
 
-
 setup(
     name='vulyk',
     version='0.3.1',
```
Comment (dchaplinsky) [inline setup.py]: Bump dat shit!

File: vulyk/tasks/__init__.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # -*- coding=utf-8 -*-
```
Comment (dchaplinsky) [inline vulyk/tasks/__init__.py]: Should we drop those lines as well?

**Comment by dchaplinsky** [comment]
Bump the version and merge dat puppy.

**Comment by mrgambal** [comment]
@dchaplinsky here we go

**Comment by dchaplinsky** [comment]
Fuck yeah.

**Comment by mrgambal** [comment]
Nicey-nicey-nice!  Time for a little of the old, ultra violence.

**Pull request merged**